test_functions/TestAudioDevice.cpp

```diff
@@ -0,0 +1,143 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio
+** File description:
+** TestMusic
+*/
+
+#include "raylib.h"
+#include <stdlib.h>
+
+#include "../src/AudioDevice.hpp"
+
+#define MAX_CIRCLES  64
+
+typedef struct {
+    Vector2 position;
+    float radius;
+    float alpha;
+    float speed;
+    Color color;
+} CircleWave;
+
+void testMusic(void)
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    SetConfigFlags(FLAG_MSAA_4X_HINT);  // NOTE: Try to enable MSAA 4X
+
+    InitWindow(screenWidth, screenHeight, "raylib [audio] example - module playing (streaming)");
+
+    //InitAudioDevice();                  // Initialize audio device
```

Comment:
I think you forgot to test your class, or at least it's not used in the test file